packages/next/src/server/app-render/app-render.tsx

```diff
@@ -1121,7 +1131,7 @@ async function renderToHTMLOrFlightImpl(
     // if PPR is enabled
     renderOpts.experimental.ppr &&
     // and a call to `postpone` happened
-    staticGenerationStore.postponeWasTriggered &&
+    response.postponeWasTriggered &&
```

Comment:
By observing the value from the returned render response, we're ensuring that if the render errors, and an error is rendered, that this condition is not hit to ensure we don't incorrectly error during render.